assets/src/scripts/selectors/flood-data-selector.js

```diff
@@ -53,3 +83,16 @@ export const getFloodGageHeightStageIndex= createSelector(
     }
 );
 
+/*
+ * Provides a function which returns the Waterwatch Flood Levels
+ */
+export const getWaterwatchFloodLevels = createSelector(
+    getWaterwatchActionStage,
+    getWaterwatchFloodStage,
+    getWaterwatchModerateFloodStage,
+    getWaterwatchMajorFloodStage,
+    (actionStage, floodStage, moderateFloodStage, majorFloodStage) => {
+        return [actionStage, floodStage, moderateFloodStage, majorFloodStage];
```

Comment:
I'm wondering why you returned flood levels as an array rather just return as. The caller of getWaterwatchFloodLevels must then know what the order is. Would it make the code more self documenting if an object was used? I think as I have said elsewhere (probably in the store/flood-inundation module), I think just storing the entire object as is in floodData in a floodLevel property would be more self documenting and make the code simpler.

Reply:
Changed selector to return as object with properties instead of array.  Sites property of call returns array so not returning exact full object because that would run in to the same array issue as you mentioned.